.eslintignore

```diff
@@ -0,0 +1,2 @@
+config/
+database/
```

Comment:
extra line please at the end of this files

Reply:
@kimotho-njoki the line is now added